<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorWidget</class>
 <widget class="QWidget" name="GeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QScrollArea" name="stepScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="stepScrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>162</width>
        <height>264</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QScrollArea" name="zoneDataScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="zoneDataScrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>162</width>
        <height>263</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Largeur</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Longeur</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="launchButton">
     <property name="text">
      <string>Générateur, go !</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="xMaxSpin">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="yMaxSpin">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
